Clients/Xamarin.Interactive.Client/Client/ViewControllers/NewWorkbookViewController.cs

```diff
@@ -53,8 +54,11 @@ public NewWorkbookViewController ()
                 select new NewWorkbookItem (appGroup.First ().Icon, appGroup.Key, appGroup)
             ).ToList ());
 
-            if (Items.Count > 0)
-                SelectedItem = Items [0];
+            var lastUsedAgentType = Prefs.UI.LastUsedAgentType.GetValue ();
+            if (lastUsedAgentType == AgentType.Unknown && Items.Count > 0)
+                selectedItem = Items [0];
+            else
+                SelectedAgentType = lastUsedAgentType;
```

Comment:
This intentionally uses the property rather than duplicating the logic from the setter.